### ISSUE TITLE
Inconsistent line endings

### ISSUE THREAD

**Issue by Jayman2000** [DescentDevelopers/Descent3#129]
Some files in this repo use LF line endings, and some files in this repo use CRLF line endings. [I created a script that will calculate how many files use LF line endings and how many files use CRLF line endings.](https://gist.github.com/Jayman2000/f8e1d010401c0f19125729546fac199b) Here’s what the script says when you run it on the most recent commit (07d988e194cfdaf87a86d798bb423af0ff25d096):

```
Total number of files that use LF line endings: 1,491 (86%)
Total number of files that use CRLF line endings: 62 (3%)
Total number of files: 1,724
```

It would be nice if all of the files in this repo used the same line endings. Are there any files that need to use LFs or need to use CRLFs? For example, I can see that `scripts/data/linuxfullhog/d3i.str` uses CRLFs. Would using LFs in that file break anything?

**Comment by GravisZro** [comment]
This is fixed in PR #145 with a bunch of text file encodings as well which prevented them from being edited on my machine.

**Comment by JeodC** [comment]
It is simple to add a `.gitattributes` file to take care of this, but scripts may depend on CRLF endings to function. It would need to be tested on the other platforms.
